release-builder.sh

```diff
@@ -50,25 +49,159 @@ STATIC_CONTAINER=$(docker ps -a | grep $CONTAINER_NAME |wc -l)
 if false; then
   echo "Static container, execute in static container to have max. cache"
   docker start $CONTAINER_NAME
-  docker exec -i $CONTAINER_NAME /hbb_exe/activate-exec bash -x /io/build-core.sh "$GITHUB_REPOSITORY" "$GITHUB_SHA" "$BUILD_CORES" "$GITHUB_RUN_NUMBER"
+  docker exec -i $CONTAINER_NAME /hbb_exe/activate-exec bash -c "source /opt/rh/gcc-toolset-11/enable && bash -x /io/build-core.sh '$GITHUB_REPOSITORY' '$GITHUB_SHA' '$BUILD_CORES' '$GITHUB_RUN_NUMBER'"
   docker stop $CONTAINER_NAME
 else
   echo "No static container, build on temp container"
   rm -rf release-build;
   mkdir -p release-build;
 
+  docker volume create cache-volume
+
+  # Create inline Dockerfile with environment setup for build-full.sh
+  DOCKERFILE_CONTENT=$(cat <<'DOCKERFILE_EOF'
+FROM ghcr.io/phusion/holy-build-box:4.0.1-amd64
+
+ARG BUILD_CORES=8
```

Comment:
It's not really a change here, but "while you're at it", it would be good if this script honoured the BUILD_CORES env, currently if you're not running as a Github Action it's hard coded to 8, which is very wasteful on bigger machines. 

```
BUILD_CORES=$(echo "scale=0 ; `nproc` / 1.337" | bc)

if [[ "$GITHUB_REPOSITORY" == "" ]]; then
  #Default
  BUILD_CORES=8
fi
```

Reply:
This setting respects the existing release-builder.sh processing.
How about removing the BUILD_CORE setting processing for cases that are not GitHub Actions?

If you want to build with any number of cores, you can replace the line `BUILD_CORES=$(echo “scale=0 ; `nproc` / 1.337” | bc)`.

Reply:
I'm not really fussed however you do it

I just built on an ubuntu 24.04 machine with many more cores, and didn't realize until a bit of the way through

Reply:
> even BUILD_CORES=${BUILD_CORES:-8}
 
Bamn, build cores as arg destroys docker caching layer btw :/